docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -173,783 +194,134 @@ To add an instance type, do the following:
 
-##### Preprovision nodes manually
+#### Provision nodes manually using the pre-provisioning script
 
-This process carries out all provisioning tasks on the database nodes which require elevated privileges. After the database nodes have been prepared in this way, the universe creation process from YugabyteDB Anywhere will connect with the nodes only via the `yugabyte` user, and not require any elevation of privileges to deploy and operate the YugabyteDB universe.
+This step is only required if you set **Manually Provision Nodes** to true on your on-prem provider configuration, and the SSH user has sudo privileges which require a password.
 
-Physical nodes (or cloud instances) are installed with a standard CentOS 7 server image. The following steps are to be performed on each physical node, prior to universe creation:
+{{< note title="Note" >}}
+If the SSH user does not have any sudo privileges, you can't use the script and need to set up the database nodes manually. Refer to [Set up on-premises nodes manually](../on-premises-manual/).
+{{< /note >}}
 
-1. Login to each database node as a user with sudo enabled (the `centos` user in centos7 images).
+To provision your nodes you can run the pre-provisioning script. The script is displayed under **Instances** on the **Instances** tab of the on-prem configuration you created.
 
-1. Add the following line to the `/etc/chrony.conf` file:
+You can manually provision each node using the pre-provisioning Python script, as follows:
 
-    ```text
-    server <your-time-server-IP-address> prefer iburst
-    ```
+1. Log in to the YugabyteDB Anywhere virtual machine via SSH.
 
-    Then run the following command:
+1. Access the Docker `yugaware` container, as follows:
 
     ```sh
-    sudo chronyc makestep   # (force instant sync to NTP server)
-    ```
-
-1. Add a new `yugabyte:yugabyte` user and group with the default login shell `/bin/bash` that you set via the `-s` flag, as follows:
-
-    ```bash
-    sudo useradd -s /bin/bash --create-home --home-dir <yugabyte-home> yugabyte  # (add user yugabyte and create its home directory as specified in <yugabyte-home>)
-    sudo passwd yugabyte   # (add a password to the yugabyte user)
-    sudo su - yugabyte   # (change to yugabyte user for execution of next steps)
+    sudo docker exec -it yugaware bash
     ```
 
-    `yugabyte-home` is the path to the Yugabyte home directory. If you set a custom path for the yugabyte user's home in the YugabyteDB Anywhere UI, you must use the same path here. Otherwise, you can omit the `--home-dir` flag.
+1. Copy and paste the Python script command from the YugabyteDB Anywhere UI. Set the flags for the command as follows:
 
-    Ensure that the `yugabyte` user has permissions to SSH into the YugabyteDB nodes (as defined in `/etc/ssh/sshd_config`).
+    - `--ask_password` - this flag instructs the script to prompt for a password (which is required if the sudo user requires password authentication).
```

Comment:
If I understand it right , this flag (`--ask_password`) is optional, but the rest aren’t.